packages/f/ffmpeg-core.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "ffmpeg-core",
+  "filename": "ffmpeg-core.js",
+  "description": "A pure Webassembly / Javascript port of FFmpeg.",
+  "keywords": [
+    "ffmpeg"
+  ],
+  "license": "LGPL-2.0",
+  "autoupdate": {
+    "source": "npm",
+    "target": "@ffmpeg/core",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*.js",
+          "**/*.js.map",
+          "**/*.wasm"
```

Comment:
```suggestion
          "*.js?(.map)",
          "*.wasm"
```
As with your other PR, we like to avoid using globstars in configs

Reply:
> In `@ffmpeg/ffmpeg@0.12.0-alpha.1`, they changed the way they packaged it, which caused the file paths to change.
>
> _After:_
>
> ![](https://user-images.githubusercontent.com/47095648/221442742-1a5582a4-acc1-416c-a932-cacec0528f4e.png)
>
> _Before:_
>
> ![](https://user-images.githubusercontent.com/47095648/221442756-3aad67c4-b967-4531-a5fa-c79afef245df.png)

-- https://github.com/cdnjs/packages/pull/1470#discussion_r1118169185

Reply:
As with #1470, if you need multiple directories, you can use a list: `{,umd/,esm/}*.js`